stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+                style="background-image: url(http://lorempixel.com/460/620/people/7/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe">    
+            </div>
+        
+            <div dir="rtl" class="fd-feed-list__content">
+                <p dir="rtl" class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <span>Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.</span>
+                </p>
+                
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>          
+        </div>
+        
+        <div class="feed-list__actions">
+            <div class="fd-popover">
+                <div class="fd-popover__control">
+                    <button class="fd-button fd-button--transparent" aria-controls="pQqQR213" aria-haspopup="true" aria-expanded="true" aria-label="More">
+                        <i class="sap-icon--overflow"></i>
+                    </button>
+                </div>
+                <div class="fd-popover__body fd-popover__body--right" aria-hidden="false" id="pQqQR213">
+                    <nav class="fd-menu">
+                        <ul class="fd-menu__list fd-menu__list--no-shadow">
+                            <li class="fd-menu__item">
+                                <a href="#" class="fd-menu__link">
+                                    <i class="sap-icon sap-icon--edit fd-feed-list__thumb fd-feed-list__icon"></i>
+                                    <span class="fd-menu__title fd-feed-list__icon-title">Edit</span>
+                                </a>
+                            </li>
+                            <li class="fd-menu__item">
+                                <a href="#" class="fd-menu__link">
+                                    <i class="sap-icon sap-icon--delete fd-feed-list__thumb fd-feed-list__icon"></i>
+                                    <span class="fd-menu__title fd-feed-list__icon-title">Delete</span>
+                                </a>
+                            </li>
+                            <li class="fd-menu__item">
+                                <a href="#" class="fd-menu__link">
+                                    <i class="sap-icon sap-icon--share-2 fd-feed-list__thumb fd-feed-list__icon"></i>
+                                    <span class="fd-menu__title fd-feed-list__icon-title">Share</span>
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+rtl.storyName = 'RTL';
+rtl.parameters = {
+    docs: {
+        storyDescription: 'RTL'

```

Comment:
The feed list item can display from right to left on the screen, so that it may be used internationally.